RELEASE.md

```diff
@@ -1,2 +1,6 @@
+# Release 0.2.0
+It is now possible to pass sort=False to the get_state_variables, get_derivatives, get_derived_quantities and get_variables_by_rdf methods of Model to get unsorted results.
```

Comment:
Minor suggestion: let's use the Markdown format to make the release notes look nice. I suggest a bullet for each change, and use monospace for code fragments, eg `sort=False`, `Model`